fe/fe-core/src/test/java/com/starrocks/sql/plan/LowCardinalityTest.java

```diff
@@ -1923,4 +1923,17 @@ public void testTempPartition() throws Exception {
             FeConstants.unitTestView = true;
         }
     }
+
+    @Test
+    public void testExistRequiredDistribution() throws Exception {
```

Comment:
add the same test to LowCardinalityTest2.java

Reply:
fixed